src/modules/commander/Commander.cpp

```diff
@@ -2002,7 +2130,15 @@ Commander::run()
 		}
 
 		// update manual_control_setpoint before geofence (which might check sticks or switches)
-		_manual_control_setpoint_sub.update(&_manual_control_setpoint);
+		if (_manual_control_setpoint_sub.update(&_manual_control_setpoint)) {
+			// manual control from mavlink
+			if (_manual_control_setpoint.data_source > manual_control_setpoint_s::SOURCE_RC) {
```

Comment:
This condition is confusing. I would rather have some `==` condition that shows me what we check for.